src/test/kotlin/org/koil/fixtures/MinioContainer.kt

```diff
@@ -0,0 +1,37 @@
+package org.koil.fixtures
+
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy
+
+import org.testcontainers.utility.Base58
+import java.time.Duration
+
+
+class MinioContainer(image: String = "minio/minio:edge") : GenericContainer<MinioContainer>(image) {
```

Comment:
This is rather than TestContainers?

Reply:
Oh I see https://en.wikipedia.org/wiki/MinIO
This is perfect for this use case.